src/utils/resmoke/resmoke.test.ts

```diff
@@ -20,6 +31,15 @@ describe("processResmokeLine", () => {
       "[js_test:startup_recovery_commit_transaction_before_stable_timestamp] MongoDB shell version v6.2.0-alpha-250-g46817bf"
     );
   });
+  it("should process odd resmoke lines", () => {
```

Comment:
maybe make this slightly more specific, like `"should correctly process resmoke lines where ctx uses :: format"` (not sure if that's the correct condition, but something like that)

Reply:
Good catch this is a dupe of tan above test